series/series.go

```diff
@@ -21,21 +20,27 @@ const (
 )
 
 var (
-	logger     = loggo.GetLogger("juju.juju.series")
-	HostSeries = hostSeries
-
-	seriesOnce sync.Once
-	series     string // filled in by the first call to hostSeries
+	logger = loggo.GetLogger("juju.juju.series")
```

Comment:
Perhaps move this definition to the only file that uses it (supportedseries.go)
and perhaps add a TODO to remove the globals there?